src/components/Courses/components/SearchBar/SearchBar.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Button } from '../../../../common/Button/Button';
+import Input from '../../../../common/Input/Input';
+import {
+	BUTTON_TEXT_ADD_COURSE,
+	BUTTON_TEXT_SEARCH,
+} from '../../../../constants';
+import styles from './searchBar.module.scss';
+
+const SearchBar = () => {
+	return (
+		<div className={styles.wrapper}>
+			<form
+				className={styles.form}
+				onSubmit={(event) => {
+					event.preventDefault();
+				}}
+			>
+				<Input></Input>
+				<Button text={BUTTON_TEXT_SEARCH}></Button>
```

Comment:
Search functionality is not implemented

Reply:
sorry, could find time to implement. Can we go without it?

Reply:
It will affect on the task mark, but it is your decision